common/performance-timeline-utils.js

```diff
@@ -42,3 +42,10 @@ function test_entries(actualEntries, expectedEntries) {
     }
   })
 }
+
+function delayedLoadListener(callback) {
+  window.addEventListener('load', function() {
+    // TODO(cvazac) Remove this setTimeout when spec enforces sync entries.
```

Comment:
Can you file a bug for this to track it?

Reply:
https://github.com/web-platform-tests/wpt/issues/13874